esbmc_ai/esbmc_util.py

```diff
@@ -1,12 +1,16 @@
 # Author: Yiannis Charalambous
 
 import os
+import re
 import sys
 from subprocess import PIPE, STDOUT, run, CompletedProcess
 from pathlib import Path
+import esbmc_ai.config as config
 from typing import Optional
-
 from . import config
+import nltk
+from nltk.tokenize import word_tokenize,sent_tokenize
+nltk.download('punkt')
```

Comment:
This downloads a library to your system without explicit user approval. This is bad design, there should be an option in the config to configure where it downloads this. We don't want to be downloading files to the user's systems without them knowing ;)